spill/snake.ino

```diff
@@ -77,8 +79,36 @@ void drawGrid() {
 }
 
 void moveBlock() {
+  //generating new pos for snake head
   int oldX = xPos;
   int oldY = yPos;
+
+
+if (growing)
+{
+  int newPositions[size+1][2];
+  //opying last element to new last
+  newPositions[size+1][0] = positions[size][0];
+  newPositions[size+1][1] = positions[size][1];
+  for (int i = size-1; i > 0; i--)
+  {
+      newPositions[i+1][0] = positions[i][0];
+      newPositions[i+1][1] = positions[i][1];
+  }
+  int positions = newPositions;
```

Comment:
setting an int to an array will give you just the first in in the first element of the list.

`positions = newpositions` if that doesn't work im not quite sure what to do. Will have to try